allure-commandline/pom.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>allure-core</artifactId>
+        <groupId>ru.yandex.qatools.allure</groupId>
+        <version>1.4.17-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>allure-commandline</artifactId>
+    <name>Allure Commandline</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>airline</artifactId>
+            <version>0.6</version>
+        </dependency>
+        <dependency>
+            <groupId>com.squareup.retrofit</groupId>
+            <artifactId>retrofit</artifactId>
+            <version>1.9.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-exec</artifactId>
+            <version>1.3</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
you are using `commons-lang` only in `ReportCommand` for method `isEmpty`. So you can remove this dep and change code to something like this:

```java
@Option(name = {"-o", "--report-dir", "--output"}, type = OptionType.COMMAND,
        description = "The directory to generate Allure report into.")
protected String reportDirectory = "allure-report";

/**
 * The string representation of path to the report directory.
 */
protected String getReportDirectory() {
    return reportDirectory;
}
```